test/select.spec.js

```diff
@@ -636,6 +639,31 @@ describe('ui-select tests', function () {
     expect(el.find('.select2-search-choice-close').length).toEqual(1);
   });
 
+  it('should clear selection (with object as source)', function() {
```

Comment:
Could you maybe make use of `createUISelect` and /or `createUIMultipleSelect` function in the tests?